### PR TITLE
perf: varnish cache with cookies

### PR DESCRIPTION
**Pull request by soyuka**

### CODE REVIEW

File: api/_docker/varnish/conf/default.vcl [hunk 1]
```diff
@@ -99,18 +101,114 @@ sub vcl_recv {
...
+  if (req.url ~ "\#") {
+    set req.url = regsub(req.url, "\#.*$", "");
+  }
+
+  # Strip a trailing ? if it exists
+  if (req.url ~ "\?$") {
+    set req.url = regsub(req.url, "\?$", "");
+  }
+
+  #if (req.http.Cache-Control ~ "(?i)no-cache") {
```
Comment (silverbackdan) [inline api/_docker/varnish/conf/default.vcl:131]: Will this need to be completed before merging to properly allow for cache invalidation?

Comment (silverbackdan) [reply api/_docker/varnish/conf/default.vcl:131]: Perhaps this is covered by line 105-119?

File: api/.env [hunk 2]
```diff
@@ -36,7 +36,7 @@ DATABASE_URL=postgres://api-platform:CHANGE_ME@db/api?server_version=12
 ###< doctrine/doctrine-bundle ###
 
 ###> nelmio/cors-bundle ###
-CORS_ALLOW_ORIGIN=^https?://(localhost|127\.0\.0\.1)(:[0-9]+)?$
+CORS_ALLOW_ORIGIN=^https?://(localhost|127\.0\.0\.1|cwa.local)(:[0-9]+)?$
```
[the remark this answers is not in the document]
Comment (soyuka) [reply api/.env:39]: but it didn't worked for docker as we use only this file in the docker compose

[the remark this answers is not in the document]
Comment (silverbackdan) [reply api/.env:39]: I think part of the setup should be to customise your .env from the .env.dist or perhaps just ignore this in the git repo. Anything personal to our setup for the project shouldn't really be in this repo.

[the remark this answers is not in the document]
Comment (silverbackdan) [reply api/.env:39]: I can merge and sort this out though

**Pull request merged**